packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -493,27 +493,29 @@ class OverflowMenu extends Component {
     );
 
     const menuBody = (
-      <ul className={overflowMenuOptionsClasses} tabIndex="-1" role="menu">
+      <ul
+        className={overflowMenuOptionsClasses}
+        tabIndex="-1"
+        role="menu"
+        aria-label={ariaLabel}>
```

Comment:
```suggestion
        aria-label={ariaLabel || this.props['aria-label']}>
```

should we also pass in the standard aria attribute as an option?